sensum-ethereum/test/legacy-sensations.json

```diff
@@ -0,0 +1,13636 @@
+[
```

Comment:
Suggestion: 
1. We could add a `legacy-sensations.md` and explain the proccess or steps to follow in order to get and format this file (legacy-sensations.json).

2. Expressing (somewhere) the datetime of the "snapshot"  would be nice too.